### ISSUE TITLE
NPE in PieChartRenderer::drawExtras

### ISSUE THREAD

**Issue by dafanasiev** [PhilJay/MPAndroidChart#4255]
**Summary**
In version tagged `v3.0.3`

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object java.lang.ref.WeakReference.get()' on a null object reference
  at com.github.mikephil.charting.renderer.PieChartRenderer.drawExtras(PieChartRenderer.java:638)
  at com.github.mikephil.charting.charts.PieChart.onDraw(PieChart.java:131)
```

**Expected Behavior**
No NPE

**Possible Solution**
I think that this possible because `mDrawBitmap` not initialized in `PieChartRenderer.drawData()` because `width` && `height` equals 0:

```
    @Override
    public void drawData(Canvas c) {

        int width = (int) mViewPortHandler.getChartWidth();
        int height = (int) mViewPortHandler.getChartHeight();

        if (mDrawBitmap == null
                || (mDrawBitmap.get().getWidth() != width)
                || (mDrawBitmap.get().getHeight() != height)) {

            if (width > 0 && height > 0) {

                mDrawBitmap = new WeakReference<Bitmap>(Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_4444));
                mBitmapCanvas = new Canvas(mDrawBitmap.get());
            } else
                return;
        }
```

Do we really need do anything in `drawExtras` when `mDrawBitmap` is null?

**Comment by MGaetan89** [comment]
A fix was merged in #3207

**Comment by almic** [comment]
Should be fixed in the latest commit, until a new release is created you can just use the latest `-SNAPSHOT` version like this

```gradle
dependencies {
    implementation 'com.github.philjay:mpandroidchart:-SNAPSHOT'
}
```
